src/Chunk.cpp

```diff
@@ -741,6 +767,29 @@ void cChunk::MoveEntityToNewChunk(cEntity * a_Entity)
 
 
 
+void cChunk::UpdateCanUnload()
```

Comment:
As stated earlier, I've been working under the assumption that incremental counters are always better because performance matters. CanUnloadCount is called once per tick per world.